resources/definitions/weefun_tina2s.def.json

```diff
@@ -0,0 +1,81 @@
+{
+   "name":"WEEFUN TINA2S",
+   "version":2,
+   "inherits":"weedo_base",
+   "metadata":{
+      "visible":true,
+      "author":"WEEFUN",
+      "manufacturer":"WEEFUN",
```

Comment:
Shouldn't this bee "Weedo"? We already have a weedo printer (the weedo x40).

When testing this in Cura, I noticed that right now there are two categories "WEEFUN" and "Weedo". I'm a bit confused by the naming, as I've been able to find both the "weedo tina2" and the "WEEFUN tina2"

Reply:
The WeeDo x40 is their bigger more traditional printer with a much bigger build platform. The WeeFun and WeeDo names are different badged printers. The Tina2 printer doesn't contain a heated bed and is very small as it is more of a starter printer for kids, or to be on your work desk at that size. The Tina2S is an upgraded version with a heated bed and the firmware is a bit more restrictive as to the files it will print. The starting GCODE must contain "MachineType:TINA2S" at the beginning of the start code or it just will not print and reject the file all together.